ndc-lambda-sdk/src/util.ts

```diff
@@ -29,3 +29,7 @@ export function getFlags(flagsEnum: Record<string, string | number>, value: numb
           : []
     });
 }
+
+export function sleep(ms: number): Promise<void> {
+  return new Promise((r) => setTimeout(r, ms))
+}
```

Comment:
What's this used for again?

Reply:
It's used in the tests to stagger function execution to test that parallelism is actually happening.

Reply:
👍